dotenv.cabal

```diff
@@ -91,7 +91,7 @@ library
   build-depends:         base >=4.7 && <5.0
                        , base-compat >= 0.4
                        , directory
-                       , megaparsec >= 7.0.1 && < 8.0
+                       , megaparsec >= 7.0.1 && < 9.0
```

Comment:
@Jagl257 This looks good. Could you please update the cabal version (e.g. `0.8.0.3`) and the changelog so that we can upload this change to hackage?